install/kubernetes/helm/istio/charts/galley/templates/deployment.yaml

```diff
@@ -11,6 +11,10 @@ metadata:
     istio: galley
 spec:
   replicas: {{ .Values.replicaCount }}
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
```

Comment:
The default `maxUnavailable` is 1 for apiVersion `extensions/v1beta1`. We should consider using `v1beta1/apps`